app/jobs/whats_app_adapter/create_templates.rb

```diff
@@ -79,6 +79,7 @@ def fetch_client_info
       waba_account = client_hash['waba_account']
       organization.update!(three_sixty_dialog_whats_app_template_namespace: waba_account['namespace'],
                            three_sixty_dialog_client_waba_account_id: waba_account['id'])
+      waba_account
```

Comment:
I've put a `binding.pry` here and it seems this code is not covered by any spec.

I would :heart: if we could test this method.